src/IAction.java

```diff
@@ -0,0 +1,6 @@
+
+public interface IAction {
+	int payment = 300;
+	public void MonthlyBill();
```

Comment:
follow the naming conventions - method name should start in lowerCase
please fix here and in the rest of the files